src/content/blog/astro-ecosystem-fund-update.mdx

```diff
@@ -0,0 +1,41 @@
+
+Astro is the web framework for building content-driven websites like blogs, marketing, and e-commerce. This year we created the Astro Ecosystem Fund to award $100,000 in large grants to ten project maintainers over the course of the year. Each recipient receives a one-time grant with no strings attached, to support and continue their work on behalf of the larger Astro ecosystem. 
+
+I'm excited to announce two new recipients in our latest fall batch:
+- **[@hippotastic](https://github.com/hippotastic)** ($10,000) for work on the [Expressive Code](https://expressive-code.com/) advanced syntax highlighter.
+- **[@martrapp](https://github.com/martrapp)** ($5,000) for work on [VTBOT](https://github.com/martrapp/astro-vtbot) exploring the bleeding edge of View Transitions in Astro and the larger web platform.
+
+We are thankful to our own major open-source sponsors for making this possible including [Netlify](https://www.netlify.com/), [Sentry](https://www.sentry.com/), [Jetbrains](https://www.jetbrains.com/) [and more](https://github.com/withastro/astro#sponsors). Without their support, this initiative would never have been possible.
+
+## State of OSS Funding
+
+Hippo and Martin are the fifth and sixth grant recipients from the fund. They join the maintainers of [Lucia](https://lucia-auth.com/), [Volar](https://volarjs.dev/), [`astro-eslint-parser`](https://github.com/ota-meshi/astro-eslint-parser) and [Lexington themes](https://lexingtonthemes.com/) who all received grants earlier this year. We have $45,000 remaining in the fund, which leaves us with at least 4 more grants to give out before the end of the year. We're already preparing for an exciting final round of announcements!
+
+Since we announced this program last December, we've also seen more attention than ever shift to the unbalanced relationship between free, open source software and the closed, for-profit corporations that benefit the most from them. 
+
+![open source pledge, illustration is of a hand stuffing money into the top of an old school chunky computer like a piggy bank](./_images/astro-ecosystem-fund-update/pledge.png)
+
+Sentry -- one of our own sponsors -- recently announced the [Open Source Pledge](https://osspledge.com/) which aims to set a basic bar for ethical corporate consumption of open source: $2,000 paid to open source software maintenance, per employee, per year. We're excited by the early momentum of this project and the larger industry conversation it's kicking off.
+
+## About Astro
+
+Astro primarily operates as an open-source project. We follow an open governance model, an open roadmap process, and a completely open budgeting process through [Open Collective](https://opencollective.com/astrodotbuild). We have a company that supports the project with full time resources, but otherwise all of our funding on Open Collective comes from sponsorships and donations to the project by companies and some amazing individuals.
```

Comment:
comma and hyphen nit
```suggestion
Astro primarily operates as an open-source project. We follow an open governance model, an open roadmap process, and a completely open budgeting process through [Open Collective](https://opencollective.com/astrodotbuild). We have a company that supports the project with full-time resources, but otherwise, all of our funding on Open Collective comes from sponsorships and donations to the project by companies and some amazing individuals.
```